.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "themes/plgd"]
 	path = themes/plgd
 	url = git@github.com:plgd-dev/www-theme.git
+[submodule "themes/hugo-algolia"]
```

Comment:
@Lukaasko is this submodule s needed ?

Reply:
after latest changes it is not needed - I deleted it